static/scss/answers/templates/vertical-grid.scss

```diff
@@ -57,12 +57,25 @@
       position: relative;
     }
 
+    &-resultsLeftContainer {
+      flex: 1;
```

Comment:
if we're just setting one flex property would it make sense to just set it directly (or to specify all 3 values)? I vaguely remember reading something against using the flex shorthand like this https://mastery.games/post/flex-shorthand/ (not this article but similar) also some weirdness with ie11? (https://stackoverflow.com/questions/21600345/flexbox-and-internet-explorer-11-displayflex-in-html)

Reply:
That make sense. Since the default value and the interpretation of flex one may be different per browser, I will set it explicitly in the flex property as 1 1 0.